README.md

```diff
@@ -113,10 +113,14 @@ docker pull openwisp/openwisp-websocket:latest
 
 5. When the containers are ready, you can test them out by going to the domain name that you've set for the modules.
 
-Note:
+**Note:**
    - Default username & password are `admin`.
-
-**Note:** If you are using pipenv, remember that changing the values in `.env` file does nothing because `.env` is also a special file in `pipenv`, you need to change the values in `.env` file then re-activate environment to ensure that the changes reflect.
+   - Default domains are: dashboard.openwisp.org, controller.openwisp.org, radius.openwisp.org and topology.openwisp.org.
+   - `To reach the dashboard` you should add the domains in `/etc/hosts` file by command 
+   ```bash
+   bash -c 'echo "127.0.0.1 <your-dashboard-domain> <your-controller-domain> <your-radius-domain> <your-topology-domain>" >> /etc/hosts'
+   ```
+   - If you are using pipenv, remember that changing the values in `.env` file does nothing because `.env` is also a special file in `pipenv`, you need to change the values in `.env` file then re-activate environment to ensure that the changes reflect.
```

Comment:
This is also a **Note** not a part of the procedure. :smile:

Reply:
I think we shouldn't `Disabling Services`, but we may create a link to it from notes, what do you think? (Guess it will be strange if we will put so much text and bullets in our `notes` part)

Reply:
Oh, sorry my bad, I thought the identation here is wrong, but it's fine! Leave it!